SwiftGit2/Repository.swift

```diff
@@ -328,12 +328,12 @@ final public class Repository {
 	/// name - The name of the remote.
 	///
 	/// Returns the remote if it exists, or an error.
-	public func remoteWithName(_ name: String) -> Result<Remote, NSError> {
+	public func remote(withName name: String) -> Result<Remote, NSError> {
```

Comment:
I'd drop this `with` too. I don't think it works well as part of an argument name in Swift.

```swift
public func remote(name: String) -> Result<Remote, NSError> {
```

This is admittedly a somewhat unusual API as far as Swift goes since all the methods like this are located on `Repository`. That makes the naming less clear.